deploy/vagrant/provision/roles/ufs_hadoop2/files/start.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# TODO comment
+# rm -rf /tmp/hadoop-ec2-user/dfs/name
```

Comment:
I used this for debugging. Without this line, running "./create 2 aws" second time would hang here.